src/SqlPersistence/SqlPersistenceStorageSessionExtensions.cs

```diff
@@ -4,6 +4,9 @@
 
 namespace NServiceBus
 {
+    using System.Data.Common;
+    using System.Threading.Tasks;
+    using Extensibility;
 
     /// <summary>
     /// Shared session extensions for NHibernate persistence.
```

Comment:
https://github.com/Particular/NServiceBus.Persistence.Sql/commit/63ae3486c799c9219fff386d103bb6c37046f4d8

and will be included when i do a cleaner PR